unattended_installer/install_functions/checks.sh

```diff
@@ -147,6 +147,71 @@ function checkArguments() {
 
 }
 
+function checkFirewalls() {
+
+
+    firewallsList=( "iptables"
+                    "nft"
+                    "ufw"
+                    "firewall-cmd")
+    
+    portsTCPLists=( "1514"
+                    "1515"
+                    "1516"
+                    "514"
+                    "55000"
+                    "9200"
+                    "9300"
+                    "9400"
+                    "443")
+
+    for command in "${firewallsList[@]}"; do
+
+        if [ -n "$(command -v $command)" ]; then
+            logger_cert "The $command command is present on this system. This could affect the correct communication between Wazuh components. We will proceed to try to validate firewall rules that may affect the processes and report what is found."
+            firewallstatus='true'
+
+            if [ $command == "iptables" ]; then
+                logger "iptables report:"
+                for port in "${portsTCPLists[@]}"; do
+                    if [ -n "$($command -L -n | grep DROP | grep $port)" ]; then
+                        logger "                 ...port $port must be open in your firewall rules."
```

Comment:
`-d` logger needed. And why this logger format? 
In addition to this, `must be opened` instead of `must be open`. 
In addition to this, one port to verify is `514`. This one is only for syslog, there is no need to check it. Ok for the other ones, but, if the user has a `91514` port blocked, this message will appear. The `grep` must be accurate. [Try `$` for ending and `^` for starting string](https://www.webservertalk.com/grep-exact-match-use-regex). 
Finally, please attach examples of behavior with the firewall configured for blocking one of the defined ports and one of a similar port.

Reply:
Done:
  - must be opened
  - grep must be accurate
  - examples of behavior

Reply:
The log format had seemed appropriate to me, now I corrected it to one line. As in the case of checkTools.